#Message where each sub command from simulink or mqs_autoRelease for the MQS is published into

uint8[16] auto_ctrls
